API.Client/AdminDlcsClient.cs

```diff
@@ -0,0 +1,68 @@
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using API.Client.JsonLd;
+using DLCS.Web.Response;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace API.Client
+{
+    public class AdminDlcsClient
+    {
+        private readonly ILogger<AdminDlcsClient> logger;
+        private readonly HttpClient httpClient;
+        private readonly JsonSerializerSettings jsonSerializerSettings;
+        
+        public AdminDlcsClient(
+            ILogger<AdminDlcsClient> logger,
+            HttpClient httpClient)
+        {
+            this.logger = logger;
+            this.httpClient = httpClient;
+            jsonSerializerSettings = new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore,
+                ContractResolver = new CamelCasePropertyNamesContractResolver()
+            };
+        }
+
+        public void SetBasicAuth(string basicAuth)
+        {
+            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", basicAuth);
```

Comment:
I think you could use `DLCS.Web.Requests.HttpRequestHeadersX.AddBasicAuth(this HttpRequestHeaders headers, string base64Encoded)` here, although i'm not sure if it's worth it as it's a single line.